python/llm/src/bigdl/llm/langchain/llms/transformersllm.py

```diff
@@ -188,6 +195,13 @@ def from_model_id_low_bit(
             model = AutoModelForCausalLM.load_low_bit(model_id, **_model_kwargs)
         except:
             model = AutoModel.load_low_bit(model_id, **_model_kwargs)
+        
+        # TODO: may refactore this code in the future
+        if 'device_map' in _model_kwargs:
```

Comment:
add a arc ut to test device_map="xpu" option

Reply:
if we don't specify device map, it probably is "auto". Maybe we need to specifically check for xpu in `device_map` instead of to.(device_map)